### PR TITLE
fix: reinstated default padding to modal

### PR DESCRIPTION
**Pull request by mjashanks**
Added a default padding to Modals.

If we need custom padding, we can use `<Modal noPadding> ... </Modal>`,  the set the padding on the content

### CODE REVIEW

File: src/Modal/Modal.svelte [hunk 1]
```diff
@@ -10,6 +10,10 @@
   export let borderColor = "";
   export let hideCloseButton = false;
 
+  // by default, all modals have padding
+  // use <Modal noPadding> ... </Modal> to remove
+  export let noPadding=false;
```
Comment (shogunpurple) [inline src/Modal/Modal.svelte:15]: Would the inline `style` prop be easier/more flexible here?

`<Modal style="padding: 0;" />`

**Review by shogunpurple — APPROVED** [review]
LGTM - although style prop might make life a bit easier and use significantly less code.

File: src/Modal/Modal.svelte (hunk 1 above)
Comment (mjashanks) [reply src/Modal/Modal.svelte:15]: yes, makes more sense actually

Comment (mjashanks) [reply src/Modal/Modal.svelte]: updated to your suggestion

**Pull request merged**